File: Iniciante/1038.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        int cod = scanf.nextInt();
        int qtd = scanf.nextInt();
        float total=0;

        if(cod == 1){

            total = (float) (qtd * 4.00);
        }else
        if(cod == 2){
    
            total = (float) (qtd * 4.50);
        }else
        if(cod == 3){
    
            total = (float) (qtd * 5.00);
        }else
        if(cod == 4){
    
            total = (float) (qtd * 2.00);
        }else
        if(cod == 5){
    
            total = (float) (qtd * 1.5);
        }
    
        System.out.printf("Total: R$ %.2f\n", total);

    }
}

File: Iniciante/1038.c
#include <stdio.h>

int main(){

    int cod, qtd;
    float total;

    scanf("%d %d", &cod, &qtd);

    switch (cod){
    case 1:
        total = qtd * 4.00;
        break;

    case 2:
        total = qtd * 4.50;
        break;

    case 3:
        total = qtd * 5.00;
        break;

    case 4:
        total = qtd * 2.00;
        break;

    case 5:
        total = qtd * 1.5;
        break;
    }

    printf("Total: R$ %.2f\n", total);

    return 0;

}